### PR TITLE
Add a list of all notable contributors

### PR DESCRIPTION
**Pull request by EmreTech**
This adds a list of all notable contributors to the t2linux project.

### CODE REVIEW

File: docs/notable-contributors.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+This page contains all Notable Contributors for the Linux on T2 project. \
+If there is someone missing that you think should be on this list, send a PR on the [wiki repository.](https://github.com/t2linux/wiki)
+
+- aunali1 (Arch Linux and Kernel Module work) [https://github.com/aunali1](https://github.com/aunali1)
+- JPyke3 (Manjaro) [https://github.com/JPyke3](https://github.com/JPyke3)
+- marcosfad (Ubuntu) [https://github.com/marcosfad](https://github.com/marcosfad)
+- MCMrARM (MacBook Bridge / T2 Linux Driver) [https://github.com/MCMrARM](https://github.com/MCMrARM)
+- mikeeq (Fedora) [https://github.com/mikeeq](https://github.com/mikeeq)
+- networkException (Wiki maintenance and `mbpfan` fork for T2) [https://github.com/networkException](https://github.com/networkException)
+- Redecorating (Wiki maintenance and providing compiled kernels with Corellium patch for wifi) [https://github.com/Redecorating](https://github.com/Redecorating)
```
Comment (AdityaGarg8) [inline docs/notable-contributors.md:12]: I guess the correlium part can be removed?

Comment (EmreTech) [reply docs/notable-contributors.md]: Fixed in commit 2a97094

**Review by networkException — APPROVED** [review]

**Comment by AdityaGarg8** [comment]
Looks like we forgot marcan here.

**Comment by EmreTech** [comment]
Oh oops

**Comment by Redecorating** [comment]
Do we need to have the list in two places?

**Comment by EmreTech** [comment]
We could remove the shorter list on the landing page (index.md) and replace it with a link to all of the notable contributors

**Comment by Redecorating** [comment]
That sounds good

**Pull request merged**